case-lib/lib.sh

```diff
@@ -86,6 +86,14 @@ func_lib_disable_pulseaudio()
     # because hijack the sudo command
     sudo "sed -i '/autospawn/cautospawn = no' $PULSEAUDIO_CONFIG"
     sudo pkill -9 pulseaudio
+    sleep 1s # wait pulseaudio to be disabled
+    if [ ! "$(ps -C pulseaudio --no-header)" ]; then
+        dlogi "Pulseaudio disabled"
+    else
+        # if failed to disable pulseaudio, test case related to pulseaudio will be skipped.
```

Comment:
Let use the N/A to replace skip from now on.

Reply:
sure!